tests/unit/model/designation-test.js

```diff
@@ -54,3 +54,191 @@ test('Relationships with other models', function(assert){
   assert.equal(relationshipOrdersPackage.key, 'ordersPackages');
   assert.equal(relationshipOrdersPackage.kind, 'hasMany');
 });
+
+test('check isLocalOrder', function(assert){
+  const model = this.subject({detailType: 'LocalOrder'});
+  assert.equal(model.get('isLocalOrder'), true);
+});
+
+test('check dispatchedItems returns items with sentOn not null', function(assert){
+  assert.expect(3);
+  const model = this.subject();
+  var store = this.store();
+  var item1 = null;
+  var item2 = null;
+  var item3 = null;
+  var despatchedItemsIds = null;
+
+  Ember.run(function(){
+    store.createRecord('item', { id: 1, sentOn: "12/07/2016" });
+    store.createRecord('item', { id: 2, sentOn: "12/07/2016" });
+    store.createRecord('item', { id: 3, sentOn: null });
+    item1 = store.peekRecord('item', 1);
+    item2 = store.peekRecord('item', 2);
+    item3 = store.peekRecord('item', 3);
+    model.get('items').pushObject(item1);
+    model.get('items').pushObject(item2);
+    model.get('items').pushObject(item3);
+  });
+  despatchedItemsIds = model.get('dispatchedItems').getEach('id');
+  assert.equal(despatchedItemsIds.get('length'), 2);
+  assert.equal(despatchedItemsIds[0], item1.get('id'));
+  assert.equal(despatchedItemsIds[1], item2.get('id'));
+});
+
+test('check designatedItems returns items with sentOn null', function(assert){
+  assert.expect(3);
+  const model = this.subject();
+  var store = this.store();
+  var item1 = null;
```

Comment:
@viveksriram1 , I don't think we need to initialise this as null, we can directly define them as var item1, item2 and it will take default value i.e undefined.
What do you think?